android/src/main/kotlin/com/difrancescogianmarco/arcore_flutter_plugin/RenderableCustomFactory.kt

```diff
@@ -26,8 +26,8 @@ class RenderableCustomFactory {
                 val url = flutterArCoreNode.objectUrl
 
                 val localObject = flutterArCoreNode.object3DFileName
-
-                if (localObject != null) {
+                Log.e(TAG, "Modified by BrutalCoding - 1");
+                if (false) {
```

Comment:
Is it a mistake?

Reply:
Hi, 

The LOG should definitly be removed and the false statement was there on purpose I believe, because localObject (fbx, GLTF, obj) were not working for me. In my personal use case, in order to use local objects, I pass an "file://path/to/something.glb" url which works. The example app you've provided with local objects also don't seem to work hence why I've set false in that condition.

Correct me if I'm wrong.

Reply:
In case you do want to merge this PR, I would actually remove localObject from Flutter's side too so people can't try it out and see nothing happening.

If localObject does work, I would be curious how. I've used Android Studio and SceneForm plugin to convert my .gltf to .sfa/.sfb but that plugin is completely broken starting from Android Studio 3.6+. I believe that plugin works with 3.5.x but never got the local object working.

Reply:
> The LOG should definitly be removed and the false statement was there on purpose I believe, because localObject (fbx, GLTF, obj) were not working for me. In my personal use case, in order to use local objects, I pass an "file://path/to/something.glb" url which works. The example app you've provided with local objects also don't seem to work hence why I've set false in that condition.
> 
> Correct me if I'm wrong

I use sfb file in my [example](https://github.com/giandifra/arcore_flutter_plugin/blob/master/example/lib/screens/assets_object.dart), The sfb files are generated from [build.gradle](https://github.com/giandifra/arcore_flutter_plugin/blob/2d25904d2c9892e9d787e4e153197198afb00115/example/android/app/build.gradle#L75)

> If localObject does work, I would be curious how. I've used Android Studio and SceneForm plugin to convert my .gltf to .sfa/.sfb but that plugin is completely broken starting from Android Studio 3.6+. I believe that plugin works with 3.5.x but never got the local object working.

Yes, the sceneform plugin doesn't work good. I'm using 1.15.0 with Android Studio 4.0.1 while Android side i'm using 1.13.0 how you can see [here](https://github.com/giandifra/arcore_flutter_plugin/blob/2d25904d2c9892e9d787e4e153197198afb00115/example/android/build.gradle#L11).

I will try different local object format inserted inside src/main/assets directly.

Thanks

Reply:
Could you remove this change and resolve conflicts? So I will accept this PR. 
We can discuss about local and asset object in another PR. I would like to edit the properties to use sfb file from assets and local object (GLTF) from path like yo do ("file://path/to/something.glb")